image_op/reconstruct_traces.m

```diff
@@ -31,8 +32,11 @@ function reconstruct_traces(movie_source, ica_dir, varargin)
 % Hakan Inan (Mar 15)
 %
 
+% Minimum number of active pixels in an IC filter to be regarded as valid
+min_num_pixels = 4;
+
 % Reconstruction parameters
-threshMov = -1; % By default keep everything
+threshMov = 0; 
```

Comment:
Can we leave `threshMov = -1`. Setting `threshMov = 0` arbitrarily removes negative values from the movie, which is not the most conservative action. You can continue to use negative removal for your own reconstructed traces.